scripts/ableplayer-base.js

```diff
@@ -68,10 +68,14 @@ exports.initAllAblePlayers = function () {
 	// Construct an AblePlayer object
 	// Parameters are:
 	// media - jQuery selector or element identifying the media.
-	window.AblePlayer = function (media) {
+	window.AblePlayer = function (media, onLoaded) {
 		// Keep track of the last player created for use with global events.
 		AblePlayer.lastCreated = this;
 		this.media = media;
+		// callback function to notify outside world that AblePlayer is fully loaded (Herbie style)
+		// how do you define "fully loaded??"
```

Comment:
😆

Reply:
once you check this in, the current AccessibleVideoWidget will not work anymore... which is fine given that there'll be a client PR coming right after this..? 😁

Reply:
yes, yes it will :)

Reply:
actually .... if should not break because if `options` is not passed by older usage thats fine as we check if `options` exists. 

Was this the concern @boronia ?

Reply:
ah yep so if we check it first then it's all good 🆗